stores/evga.py

```diff
@@ -61,17 +63,33 @@ def login(self, username, password):
             "Object.defineProperty(navigator, 'webdriver', {get: () => undefined})"
         )
 
-        self.driver.get(LOGIN_URL)
-        selenium_utils.wait_for_page(self.driver, "EVGA - Intelligent Innovation")
+        if path.isfile("cookies.pkl"):  # check for cookies file
+            self.driver.get("https://www.evga.com")
+            selenium_utils.wait_for_page(
+                self.driver, "EVGA - Intelligent Innovation - Official Website", 300
+            )
+            cookies = pickle.load(open("cookies.pkl", "rb"))
```

Comment:
could you name this evga-cookies or something similar so we can reuse this logic?

Reply:
done